circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
-    - python setup.py docs:
+    # sphinx fails with python 3.5.1 due to an issue in typing package,
+    # and node 1 has python 3.4.3
+    - if [ $CIRCLE_NODE_INDEX == 1 ] ; then pip install -r docs-requirements.txt ; fi:
+        parallel: true
+    - if [ $CIRCLE_NODE_INDEX == 1 ] ; then rm -rf docs/API && make -C docs clean ; fi:
```

Comment:
Thanks @jayvdb for your investigation and their results in https://github.com/coala/coala-bears/pull/1826#issuecomment-307840747 :) So I will rebase now, keeping the `rm ...` stuff deleted, and then finally merge this